<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardProdWindow</class>
 <widget class="QWidget" name="CardProdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Карточка изделия</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>751</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Карточка</string>
    </attribute>
    <widget class="QLineEdit" name="leDateOn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата включения</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leContract">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leObjectInstall">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leProduction">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leGarant">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>130</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата производства</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leDateCreate">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Карта заказа</string>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Продукция</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>210</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Договор</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leNumber">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="leCardOrder">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>480</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Объект установки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>130</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Годен до</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Сервисное обслуживание</string>
    </attribute>
    <widget class="QPushButton" name="pbAddRemont">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Вывести в ремонт</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbEditStatus">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить Статус</string>
     </property>
    </widget>
    <widget class="QTreeWidget" name="trwRemont">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>721</width>
       <height>351</height>
      </rect>
     </property>
     <property name="animated">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="headerHighlightSections">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>№ рекламации</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата начала</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дата окончания</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Причина</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Действие</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Дефект</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Замечание</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Статус</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Состав изделия</string>
    </attribute>
    <widget class="QListWidget" name="lwInclude">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>481</width>
       <height>421</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>В составе набора</string>
    </attribute>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>561</width>
       <height>421</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>Набор оборудования</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
